src/index.html

```diff
@@ -11,12 +11,108 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo__link"
+      >
+        <img
+          src="images/logo.png"
```

Comment:
The `alt` attribute for the logo image is 'logo'. Consider making it more descriptive, such as 'Company Logo' or 'Moyo Logo', to improve accessibility.